scripts/ci/check_pull_request.py

```diff
@@ -81,9 +81,17 @@ def main():
 def check_pull_request(title, body):
     if title.startswith('['):
         error_flag = regex_line(title)
+        history_note_error_flag = False
+        edit_history_note = False
         for line in body:
             if line.startswith('['):
-                error_flag = regex_line(line) or error_flag
+                # get component name in []
+                ref = re.findall(r'[\[](.*?)[\]]', line)
+                if ref and ref[0] not in ['Component Name 1', 'Component Name 2']:
+                    edit_history_note = True
+                    history_note_error_flag = regex_line(line) or history_note_error_flag
+		# If edit history notes, ignore title check result
+        error_flag = error_flag if not edit_history_note else history_note_error_flag
```

Comment:
If no error is reported from history notes (`history_note_error_flag` is 'False`), while there is error from title (`error_flag` is `True`), then CI will fail, seems it's not what we want.